crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1269,6 +1276,44 @@ impl<S: Eq + Debug + Serialize + for<'de> Deserialize<'de>> SuiDataStore<S> {
         Ok(batch.write()?)
     }
 
+    /// Returns transaction digests from consensus_message_order table in the "checkpoint range".
+    ///
+    /// Checkpoint range is defined from the last seen checkpoint(excluded) to the provided
+    /// to_height (included)
+    pub fn last_checkpoint(
```

Comment:
The function name is a bit confusing to me. Does this function return the list of digests since the most recent checkpoint boundary?

Reply:
Yes. I added doc/comment above the function to describe what it does, but happy to rename it too if you have suggestion